tests/test_modeling_tf_bart.py

```diff
@@ -223,33 +283,10 @@ def test_lm_uneven_forward(self):
         self.assertEqual(outputs.logits.shape, expected_shape)
 
 
-def _assert_tensors_equal(a, b, atol=1e-12, prefix=""):
-    """If tensors not close, or a and b arent both tensors, raise a nice Assertion error."""
-    if a is None and b is None:
-        return True
-    try:
-        if tf.debugging.assert_near(a, b, atol=atol):
-            return True
-        raise
-    except Exception:
-        msg = "{} != {}".format(a, b)
-        if prefix:
-            msg = prefix + ": " + msg
-        raise AssertionError(msg)
-
-
-def _long_tensor(tok_lst):
-    return tf.constant(tok_lst, dtype=tf.int32)
-
-
-TOLERANCE = 1e-4
-
-
-@is_pt_tf_cross_test
 @slow
 class TFBartModelIntegrationTest(unittest.TestCase):
     def test_inference_no_head(self):
-        model = TFBartModel.from_pretrained("facebook/bart-large", from_pt=True)
+        model = TFBartForConditionalGeneration.from_pretrained("facebook/bart-large").model
```

Comment:
Why not use `TFBartModel`?

Reply:
doesn't work, the weights don't load correctly...this didn't work previously either. Could be dealt with in a future PR